content/docs/querying/functions.md

```diff
@@ -40,6 +40,12 @@ by sample value.
 `ceil(v instant-vector)` rounds the sample values of all elements in `v` up to
 the nearest integer.
 
+## `changes()`
+
+For each input time series, `changes(v range-vector)` returns the number of
+times its value has changed within the provided time range as an instant
+vector.
```

Comment:
Should we mention that it's fitted towards integers? For floats with small changes the result is fairly unpredictable.